### PR TITLE
feat: add onboarding pagination

### LINKED ISSUE #380: wip: fix the user onboarding flow
**Tasks:**

- [ ] fix the user onboarding flow
- [ ] add a [stepper](https://mui.com/material-ui/react-stepper/)

**Screenshots**
image

### PR DESCRIPTION
**Pull request by Shurtu-gal**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes #380 

### Description

Below are the screenshots.
Screenshot from 2023-01-04 19-38-47
Screenshot from 2023-01-04 19-38-54
Screenshot from 2023-01-04 19-38-59
Screenshot from 2023-01-04 19-39-11
Screenshot from 2023-01-04 19-39-14
Screenshot from 2023-01-04 19-39-20

And this is for mobile view:-

     

❤️Thank you!

### Post merge checklist

- [x] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Review by 120EE0692 — CHANGES_REQUESTED** [review]
@Shurtu-gal can you make the onboarding experience smooth for mobile users also.

**Comment by Shurtu-gal** [comment]
> @Shurtu-gal can you make the onboarding experience smooth for mobile users also.

Done

**Review by 120EE0692 — CHANGES_REQUESTED** [review]
image
make the text inside the button inline, refer to figma.

figma:
image

**Comment by Shurtu-gal** [comment]
Done 👍🏻

File: client/src/components/onboarding/stages/Welcome.js [hunk 1]
```diff
@@ -70,27 +68,26 @@ const useStyles = makeStyles((theme) => ({
...
-    padding: 10,
   },
   logo: {
     marginTop: '48px',
   },
   welcomeText: {
-    width: '330px',
-    height: '56px',
+    maxWidth: '330px',
+    // height: '56px',
```
Comment (120EE0692) [inline client/src/components/onboarding/stages/Welcome.js:83]: @Shurtu-gal remove this if not required

**Review by 120EE0692 — CHANGES_REQUESTED** [review]

File: client/src/components/onboarding/stages/Welcome.js (hunk 1 above)
Comment (Shurtu-gal) [reply client/src/components/onboarding/stages/Welcome.js:83]: Sure

**Review by 120EE0692 — APPROVED** [review]

**Review by rutajdash — APPROVED** [review]
Great job with the pagination CSS!

**Pull request merged**